diff --git a/src/Makefile.am b/src/Makefile.am
index ff6a6d6f3..f780d6c59 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -385,18 +385,7 @@ harfbuzz.cc: Makefile.sources
 	|| ($(RM) $(srcdir)/harfbuzz.cc; false)
 BUILT_SOURCES += harfbuzz.cc

-noinst_PROGRAMS = \
-	main \
-	test \
-	test-buffer-serialize \
-	test-ot-meta \
-	test-ot-name \
-	test-ot-glyphname \
-	test-gpos-size-params \
-	test-gsub-get-alternates \
-	test-gsub-would-substitute \
-	test-use-table \
-	$(NULL)
+noinst_PROGRAMS =
 bin_PROGRAMS =

 main_SOURCES = main.cc
